tests/lax_control_flow_test.py

```diff
@@ -2044,34 +2094,58 @@ def sqrt_cubed(x, tangent_solve=scalar_solve):
 
     inputs = jnp.array([4.0, 5.0])
     results = jax.vmap(sqrt_cubed)(inputs)
-    self.assertAllClose(results, inputs ** 1.5, check_dtypes=False)
+    self.assertAllClose(results, inputs ** 1.5, check_dtypes=False, atol=1e-6, rtol=1e-6)
 
     results = jax.jit(sqrt_cubed)(5.0)
     self.assertAllClose(
         results, 5.0**1.5, check_dtypes=False, rtol={np.float64: 1e-7})
 
+  @parameterized.named_parameters([
+    {"testcase_name": "vector_solve", "solve_method": vector_solve, "modes": ["fwd", "rev"]},
+    {"testcase_name": "newton_raphson", "solve_method": newton_raphson, "modes": ["fwd"]},
+  ])
   @jtu.skip_on_flag("jax_skip_slow_tests", True)
-  def test_custom_root_vector_with_solve_closure(self):
-
-    def vector_solve(f, y):
-      return jnp.linalg.solve(jax.jacobian(f)(y), y)
-
-    def linear_solve(a, b):
-      f = lambda y: high_precision_dot(a, y) - b
-      x0 = jnp.zeros_like(b)
-      solution = jnp.linalg.solve(a, b)
-      oracle = lambda func, x0: solution
-      return lax.custom_root(f, x0, oracle, vector_solve)
-
-    rng = self.rng()
-    a = rng.randn(2, 2)
-    b = rng.randn(2)
-    jtu.check_grads(linear_solve, (a, b), order=2,
-                    atol={np.float32: 1e-2, np.float64: 1e-11})
-
-    actual = jax.jit(linear_solve)(a, b)
-    expected = jnp.linalg.solve(a, b)
-    self.assertAllClose(expected, actual)
+  def test_custom_root_vector_with_solve_closure(self, solve_method, modes):
+    with jax.disable_jit():
```

Comment:
This should be removed. (Side note: It also doesn't make a difference for disabling `while_loop` here, because `custom_root` traces the given solver into a JAXpr, which uses JIT compilation no matter what.)